data/services.json

```
@@ -62,7 +62,7 @@
     "name": "FAIR storage",
     "description": "Details about how researchers, research groups, and organisations can store their data using SciLifeLab FAIR Storage resources.",
     "url": "/services/fairstorage/",
-    "thumbnail": "/img/service_thumbnails/fairstorage.jpg",
+    "thumbnail": "/img/service_thumbnails/storage.jpg",
```

All looks good, except here it should be `fairstorage`, it is a new change after you created a branch. I believe while resolving the conflict you retained to the original version when start working ^^

```suggestion
    "thumbnail": "/img/service_thumbnails/fairstorage.jpg",
```

---

Thanks, I think I've fixed this 5 times already 😅

---

I had forgotten to push it! Fixed now